debugger/gdb-extension/gdb_sgx_plugin.py

```diff
@@ -404,12 +407,86 @@ def oe_debugger_cleanup():
         gdb.execute("remove-symbol-file -a %s" % (oe_enclave_addr), False, True)
     g_loaded_oe_enclave_addrs.clear()
     g_enclave_list_parsed = False
+    oe_debugger_cleanup_hook()
     return
 
 def exit_handler(event):
     oe_debugger_cleanup()
     return
 
+# Figure out missing symbols within enclaves.
+import shutil
+import subprocess
+
+class MissingSymbolAnalyzer:
+    def __init__(self):
+        self.libs = {}
+        # objdump is used to figure out imports and exports of
+        # a module.
+        self.enabled = shutil.which('objdump') is not None
+        if not self.enabled:
+            self.message('objdump not available. Missing symbols cannot be analyzed.')
+        else:
+            print('oegdb: objdump found. Mising symbols will be analyzed.')
+
+    def message(self, m):
+        # unresolved_color = '\033[1;38;5;170m'
```

Comment:
Remove commented line?

Reply:
Keeping the line for now. That was another color that I liked. Quite hard to figure out the color values.